linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/tests/correct_stig.pass.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# make existing entries pass
+for acct in $(grep -ve ':\(!!\|\*\):' /etc/shadow | awk -F: '{print $1}'); do
+    chage -m 1 -d $(date +%Y-%m-%d) $acct
+done
+echo 'max-test-user:$1$q.YkdxU1$ADmXcU4xwPrM.Pc.dclK81:18648:1:60::::' >> /etc/shadow
```

Comment:
The default value of `var_accounts_minimum_age_login_defs` is 7, but the value set by the test is 1.

The suggestion above forces the variable to be 1.

Reply:
I've worked around it by going with large/small values (depending on whether or not we are looking at the min or max) for the pre-remediation. If someone wants to have a policy with a max change of 1d or a min change of 300d, we can figure something else out then.

Reply:
The tests are still failing to me, the check actually checks that the user's max age is between the min and max age. So an arbitrarily small/large number will eventually be out of the min~max range.